pyartcd/pyartcd/pipelines/prepare_release_konflux.py

```
@@ -692,6 +701,149 @@ async def finalize_shipment(self, shipment_data: tuple):
+                    changed = True
+            if changed:
+                modified[kind] = shipment
+
+        if not modified:
+            self.logger.info("No advisory ID placeholders found in shipment MR YAML, skipping")
+            return
+
+        try:
+            await self.update_shipment_mr(modified, env, shipment_url)
+            self.logger.info("Resolved advisory ID placeholders in shipment MR: %s", list(modified))
+        except Exception as ex:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)